website/views/static/community-guidelines.jade

```diff
@@ -30,14 +30,14 @@ block content
         |&nbsp;
         =env.t('February')
         |&nbsp;9&comma; 2015
-      h2=env.t('commGuideHeadingWelcome')
+      h2(id="welcome")=env.t('commGuideHeadingWelcome')
       div(class='clearfix')
         img(class='pull-left', src='/community-guidelines-images/intro.png', alt='')
```

Comment:
If you want to be super awesome, you could change the things that look like `element(class='class-name')` to `element.class-name`.

That's not strictly required for this PR, but would be nice to be consistent.